packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1213,12 +1214,6 @@ export class ContainerRuntime
 		}
 	}
 
-	/**
-	 * True if we have ID compressor loading in-flight (async operation). Useful only for
-	 * this.idCompressorMode === "delayed" mode
-	 */
-	protected _loadIdCompressor: Promise<void> | undefined;
-
```

Comment:
Per the above, we no longer need to track the "loading in progress" state now that 'createIdCompressorFn' is synchronous.